DESCRIPTION

```diff
@@ -26,6 +26,7 @@ Imports:
     stats,
     data.table,
     Rcpp (>= 0.12.15),
+    mgcv,
```

Comment:
Shouldn't this be under suggests along with ranger, xgboost etc?

Reply:
Indeed! Will fix.

Reply:
Fixed with 525f46e